manifests/init.pp

```diff
@@ -59,7 +59,7 @@
 ) {
 
   package { 'katello':
-    ensure => installed,
+    ensure  => installed,
```

Comment:
```suggestion
    ensure => installed,
```

I believe this is a relic of a previous change that is no longer included. Let's simplify the diff / not add unnecessary white space